src/MarketSigner.js

```diff
@@ -253,10 +253,19 @@ function digestPayNow({ auctionId, amount }) {
   });
 }
 
-function digestCashout({ paymentId, iban }) {
+function digestCashoutFiat({
+  nonce, iban, currencyId, amount,
```

Comment:
Everywhere else in the code we're using "price" instead of "amount", and it is using a uint instead of a "string". Please consider unifying its usage. An int seems like a better approach, since it rules out doubts about whether "." or "," symbols need to be used.

Reply:
Changed string to uint256, but I kept the name amount since, this is the cashout and we are not paying a price, we are withdrawing some amount, so I thought it makes more sense.